x/ecocredit/core/msg_bridge_receive_test.go

```diff
@@ -193,7 +193,6 @@ func TestMsgBridgeReceive_ValidateBasic(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			t.Parallel()
```

Comment:
Heads up. This was causing the same test to be run multiple times and not the others.

Reply:
Also see https://github.com/regen-network/regen-ledger/pull/1230#discussion_r912325613

Reply:
Reverted in https://github.com/regen-network/regen-ledger/pull/1226/commits/70e569a47aacc96a724faca95d51bf71a3c87c58. This caused several errors unrelated to the scope of this pull request and the get and reset message functionality used within this test was not working as intended and was causing additional errors. Opening a separate pull request to address and will likely migrate to gherkin/gocuke.

Reply:
Resolved in #1231